### PR TITLE
Add Kops upgrade cluster notes for `kops validate`

### PR DESCRIPTION
**Pull request by joshmyers**
## what

Upgrading a cluster that is already not passing `kops validate` is not a good idea, we should reflect this in our docs.

screenshot 2018-12-13 at 10 55 28

### CODE REVIEW

File: content/geodesic/kops/upgrade-cluster.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ Then follow the instructions below for upgrading Kubernetes.
 <https://github.com/kubernetes/kops/blob/master/docs/releases.md>
 {{% /dialog %}}
 
+{{% dialog type="info" icon="fa fa-book" title="Note" %}}
+Before attempting to upgrade your Kops cluster, ensure it is in a good
+state by using [kops validate](https://github.com/kubernetes/kops/blob/master/docs/cli/kops_validate.md)
```
Comment (osterman) [inline content/geodesic/kops/upgrade-cluster.md]:
```suggestion
state by running [`kops validate cluster`](https://github.com/kubernetes/kops/blob/master/docs/cli/kops_validate.md)
```

File: content/geodesic/kops/upgrade-cluster.md [hunk 2]
```diff
@@ -19,6 +19,11 @@ Then follow the instructions below for upgrading Kubernetes.
 <https://github.com/kubernetes/kops/blob/master/docs/releases.md>
 {{% /dialog %}}
 
+{{% dialog type="info" icon="fa fa-book" title="Note" %}}
+Before attempting to upgrade your Kops cluster, ensure it is in a good
```
Comment (osterman) [inline content/geodesic/kops/upgrade-cluster.md]:
```suggestion
Before attempting to upgrade any Kops cluster, ensure that it is in a healthy
```

File: content/geodesic/kops/upgrade-cluster.md [hunk 3]
```diff
@@ -19,6 +19,11 @@ Then follow the instructions below for upgrading Kubernetes.
 <https://github.com/kubernetes/kops/blob/master/docs/releases.md>
 {{% /dialog %}}
 
+{{% dialog type="info" icon="fa fa-book" title="Note" %}}
```
Comment (osterman) [inline content/geodesic/kops/upgrade-cluster.md]:
```suggestion
{{% dialog type="important" icon="fa fa-exclamation-triangle" title="Important" %}}
```

**Comment by joshmyers** [comment]
@osterman comments addressed and rebased.

**Pull request merged**